### ISSUE TITLE
<!--! do not remove --> causes IE11 to not load and Edge to have warnings

### ISSUE THREAD

**Issue by stramel** [PolymerElements/polymer-starter-kit#1025]
Transferring my comment on https://github.com/Polymer/polymer-cli/issues/779 to here

### Versions & Environment

- Polymer CLI: 1.1.0
- node: 6.9.1
- Operating System: Windows 10

#### Steps to Reproduce
1. Install `polymer-cli` (`npm i -g polymer-cli)
1. Create a directory for the app and change to that directory (`mkdir test && cd $_`)
1. Initialize `polymer-2-starter-kit` (`polymer init polymer-2-starter-kit`)
1. Serve app (`polymer serve -o`)

#### Expected Results

App loads successfully in all supported browsers

#### Actual Results

App fails to load in IE11. Also, throws warnings in IE and Edge.

### Browsers Affected

- [ ] Chrome
- [ ] Firefox
- [ ] Safari 10 (Untested)
- [ ] Safari 9 (Untested)
- [x] Edge (Warnings)
- [x] IE 11 (Fails to Load and Warnings)

### Console Output
**Edge:**
![image](https://cloud.githubusercontent.com/assets/855184/26690203/0a57c4b4-46be-11e7-943a-c0ff37cdc14e.png)

**IE11:**
![image](https://cloud.githubusercontent.com/assets/855184/26690228/21ab75de-46be-11e7-8b49-ae4c41124eb4.png)

**Comment by JoceM** [comment]
Could you find out from which file that error occurs ?

**Comment by stramel** [comment]
It shouldn't be occurring anymore in the latest changes. It was an issue with `new URL` not being supported in IE11.

**Comment by Inrego** [comment]
This is also happening for me. How do I get these "latest changes"?

**Comment by stramel** [comment]
You can pull the latest of PSK from the branch 2.0-preview manually or  I believe master has the changes as well if you don't want 2.0 but don't quote me on that

**Comment by Inrego** [comment]
I tried the 2.0-preview branch, and it's still behaving kind of honky.
Here are my findings:

First I tried to apply the differences between my version and this version, to my project. All my findings below are using IE11.

*polymer serve*
 * The application loads and the interface shows.
 * If developer tools are open (F12), IE uses massive CPU (25% of a high-end CPU) and the application never loads. After a while, lots of js errors.

*ASP.NET*
 * Application doesn't load.
 * If developer tools are open (F12), same behavior as with polymer serve.

Note that the app works with no js errors with both "polymer serve", and ASP.NET when using Chrome.
